Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -138,6 +138,19 @@ bool getVideoSizeAndFormatFromCaps(const GstCaps* caps, WebCore::IntSize& size,
         pixelAspectRatioNumerator = GST_VIDEO_INFO_PAR_N(&info);
         pixelAspectRatioDenominator = GST_VIDEO_INFO_PAR_D(&info);
         stride = GST_VIDEO_INFO_PLANE_STRIDE(&info, 0);
+    } else {
+        format = GST_VIDEO_FORMAT_UNKNOWN;
+        stride = 0;
+        int width = 0, height = 0;
+        gst_structure_get_int(structure, "width", &width);
+        gst_structure_get_int(structure, "height", &height);
```

Comment:
Return values should be checked.

Reply:
Hmm, these fallbacks are based off of the `areEncryptedCaps` paths which also don't have checked returns.